pyslackersweb/tasks.py

```diff
@@ -76,19 +77,23 @@ def sync_slack_timezones(app: web.Application):
                 params = {}
                 async with session.get(
                     "https://slack.com/api/users.list",
-                    headers={"Authorization": f"Bearer {app['slack_oauth_token']}"},
+                    headers={"Authorization": f"Bearer {oauth_token}"},
                     params=params,
                 ) as r:
                     result = await r.json()
-
+                
+                logger.debug(result)
                 for user in result["members"]:
                     if user["deleted"] or user["is_bot"]:
                         continue
 
                     counter[user["tz"]] += 1
 
+                logger.debug(counter)
                 if "next_cursor" in result.get("response_metadata", {}):
                     params["cursor"] = result["response_metadata"]["next_cursor"]
+                    logger.debug("sleeping")
+                    await asyncio.sleep(10)
```

Comment:
10 Seems rather high, doesn't it? Given the [tier](https://api.slack.com/methods/channels.list) we can do [20/min](https://api.slack.com/docs/rate-limits#overview) - the sleep should be around 3sec